robottelo/config/validators.py

```diff
@@ -10,7 +10,7 @@
         ),
     ],
     server=[
-        Validator('server.hostname', default=''),
+        Validator('server.hostname', must_exist=True),
```

Comment:
we don't have any field for `settings.server.hostname` in server.yml, we set it when assigning a xdist worker from settings.server.hostnames, so we can remove `must_exist=True` from this validator, right?
https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/core/xdist.py#L45